backend/src/main/java/gov/cdc/usds/simplereport/service/UploadService.java

```diff
@@ -74,7 +77,7 @@ private static CsvSchema personSchema() {
                 .addColumn("MiddleName", CsvSchema.ColumnType.STRING)
                 .addColumn("LastName", CsvSchema.ColumnType.STRING)
                 .addColumn("Suffix", CsvSchema.ColumnType.STRING)
-                .addColumn("Race", CsvSchema.ColumnType.STRING)
+                .addColumn("Race", CsvSchema.ColumnType.ARRAY)
```

Comment:
This seems inconsistent with the change above, which assumes a single value for `race`.